selenium_tests/questionnaires_all/test_questionnaire_functionality.py

```diff
@@ -165,11 +165,116 @@ def test_survey_back(self):
         results_page.go_back()
         current_page = QuestionnairePage(self.selenium)
         self.assertIn(survey01.title, current_page.title)
-        
-        
 
+    def test_skip_question(self):
+        survey01 = SurveyFactory(
+            parent=self.section01, 
+            owner=self.user,
+            allow_multiple_submissions = True,
+            multi_step = True,
+            thank_you_text="[{\"type\": \"paragraph\", \"value\": \"<p>Thankyou for completing the survey</p>\"}]"
+            )
 
+        SurveyFormFieldFactory(
+            page=survey01,
+            pk= 1,
```

Comment:
I think it shouldn't be necessary to specify the primary key - let the database do it.

Reply:
have removed